doc/usage/tutorial.md

```diff
@@ -162,3 +208,125 @@ NOTE:
+
+```
+E0327 06:30:45.494416       1 reflector.go:134] github.com/kubernetes-sigs/kube-batch/pkg/scheduler/cache/cache.go:302: Failed to list *v1.Pod: pods is forbidden: User "system:serviceaccount:kube-system:default" cannot list pods at the cluster scope
+E0327 06:30:45.495362       1 reflector.go:134] github.com/kubernetes-sigs/kube-batch/pkg/scheduler/cache/cache.go:306: Failed to list *v1.PersistentVolumeClaim: persistentvolumeclaims is forbidden: User "system:serviceaccount:kube-system:default" cannot list persistentvolumeclaims at the cluster scope
+E0327 06:30:45.496355       1 reflector.go:134] github.com/kubernetes-sigs/kube-batch/pkg/scheduler/cache/cache.go:308: Failed to list *v1alpha1.Queue: queues.scheduling.incubator.k8s.io is forbidden: User "system:serviceaccount:kube-system:default" cannot list queues.scheduling.incubator.k8s.io at the cluster scope
+E0327 06:30:45.497695       1 reflector.go:134] github.com/kubernetes-sigs/kube-batch/pkg/scheduler/cache/cache.go:301: Failed to list *v1beta1.PodDisruptionBudget: poddisruptionbudgets.policy is forbidden: User "system:serviceaccount:kube-system:default" cannot list poddisruptionbudgets.policy at the cluster scope
+E0327 06:30:45.498641       1 reflector.go:134] github.com/kubernetes-sigs/kube-batch/pkg/scheduler/cache/cache.go:303: Failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:kube-system:default" cannot list nodes at the cluster scope
+E0327 06:30:45.499895       1 reflector.go:134] github.com/kubernetes-sigs/kube-batch/pkg/scheduler/cache/cache.go:305: Failed to list *v1.PersistentVolume: persistentvolumes is forbidden: User "system:serviceaccount:kube-system:default" cannot list persistentvolumes at the cluster scope
+
+```
+
+`kube-batch` need to collect cluster information(such as Pod, Node, CRD, etc) for scheduling, so the service account used by the deployment must have permission to access those cluster resources, otherwise, `kube-batch` will fail to startup. For users who are not familiar with Kubernetes RBAC, please copy the example/role.yaml into `$GOPATH/src/github.com/kubernetes-sigs/kube-batch/deployment/kube-batch/templates/` and reinstall batch.
+
+
+### How to enable gang-scheduling on kubeflow/tf-operator for kube-batch?
+Please follow tf-operator [quick start](https://www.kubeflow.org/docs/components/tftraining/#quick-start) to install tf-operator.
+
+1. Add following rules to `tf-job-operator` clusterrole to grant tf-job-operator permission to create `PodGroup`.
+
+```yaml
+- apiGroups:
+  - scheduling.incubator.k8s.io
+  resources:
+  - podgroups
+  verbs:
+  - '*'
+
+```
+
+2. Enable gang-scheduling in `tf-operator` by add flag `--enable-gang-scheduling=true`
+```
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: tf-job-operator
+  ....
+spec:
+.....
+    spec:
+      containers:
+      - command:
+        - /opt/kubeflow/tf-operator.v1beta2
```

Comment:
@Jeffwan I wants to confirm your env  `tf-operator  apiVersion` ? Use v0.5.0 now? 

Add `--enable-gang-scheduling=true` is use `tf-operator.v1beta2` ?  
But Example yaml use `apiVersion: "kubeflow.org/v1beta1" `.